<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecurityTab</class>
 <widget class="QWidget" name="SecurityTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Users and Groups</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="trustee_view">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="add_trustee_button">
       <property name="text">
        <string>Add...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_well_known_trustee_button">
       <property name="text">
        <string>Add well-known trustee...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_trustee_button">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Permissions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="permissions_tab_widget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="common_permissions_tab">
      <attribute name="title">
       <string>Common</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="extended_permissions_tab">
      <attribute name="title">
       <string>Extended</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="custom_permissions_tab">
      <attribute name="title">
       <string>Custom</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
